.flake8

```diff
@@ -0,0 +1,8 @@
+[flake8]
```

Comment:
We should probably be consistent for this line: https://github.com/prometheus/client_python/blob/master/tox.ini#L83

Reply:
Makes sense, done.